### PR TITLE
[7.0][ENH]field_in_product_extension

### PR DESCRIPTION
**Pull request by kranokporn**
Add field in Product Extension.

### CODE REVIEW

**Comment by kranokporn** [comment]
issue : T0929

cc. @newtratip

File: product_extension/product.py [hunk 1]
```diff
@@ -136,34 +136,36 @@ def _name_get(d):
...
     _columns = {
         'brand_id': fields.many2one('product.brand', 'Brand'),
         'model': fields.char('Model'),
         'full_name': fields.function(_get_full_name, type='char', size=256, string="Full Name",
                 store={
                        'product.product': (lambda self, cr, uid, ids, c={}: ids, ['default_code','name', 'brand_id','model'], 20)
-                    })
-    
+                    }),
+        'tariff_description': fields.char('Tariff Description', size=13,),
```
Comment (newtratip) [inline product_extension/product.py]:
```suggestion
        'tariff_description': fields.char('Tariff Description'),
```

File: product_extension/product.py [hunk 2]
```diff
@@ -136,34 +136,36 @@ def _name_get(d):
...
+        'duty_tax_rate': fields.char('Duty Tax Rate (%)', size=13,),
```
Comment (newtratip) [inline product_extension/product.py]:
```suggestion
        'duty_tax_rate': fields.char('Duty Tax Rate (%)'),
```

**Review by newtratip — COMMENTED** [review]
Reviewed.

**Pull request merged**